README.md

```diff
@@ -10,122 +10,108 @@ Android Auto for IDrive
+| **Seeing the music/media apps** ||
+|Several new entries will be added to the Media section of the control screen. The Audioplayer icon is the one with the main functionality, while the other displayed apps above the Audioplayer are quick shortcuts to switch playback to the respective apps. This screenshot also shows the official Spotify app at the bottom of the list.|![Music App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-medialist.jpg)|
+
+
+Implemented Features
+--------------------
+
+<details>
+<summary>🎙 Voice Assistants</summary>
 
-After connecting the phone to the car, the official Connected app should show this car icon in the status bar. When this icon appears, this app should connect and add its functionality to the car.
+- Any voice assistant installed on the phone is added as a Connected App entry
+- Google Assistant works the best, but Alexa, Bixby, and Cortana are also compatible
+- The voice assistant app can be set as a hardware shortcut button for convenience
 
-See [this guide](https://hufman.github.io/AndroidAutoIdrive/connection.html) for tips on improving the connection reliability of the Connected app.
+</details>
 
-![Phone App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-phoneapps.jpg)
+<details>
+<summary>🔔 Phone Notifications</summary>
 
-After all the apps are connected, a bunch of new entries will show up in the car's Connected menu. Besides the official Calendar and Connected apps, there should be a new Audioplayer icon and a book icon with no label. This book icon is the Notifications app, if enabled.
+- Popup about new notifications (on IDrive 4 or older)
+- Supports Dismiss, Mark As Read, replies or other notification actions
 
-![Music App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-medialist.jpg)
+</details>
+<details>
+<summary>🎵 Control of Android Auto compatible music apps</summary>
 
-Several new entries will be added to the Media section of the control screen. The Audioplayer icon is the one with the main functionality, while the other displayed apps above the Audioplayer are quick shortcuts to switch playback to the respective apps. This screenshot also shows the official Spotify app at the bottom of the list.
+- Supports browsing and searching apps' music libraries
+- Supports selecting from a list of currently-queued songs, as well as basic back/next control
+- Integrates into the car's audio context, for automatic resume and hardware button control
+- Supports controlling any active music session, even apps that aren't Android Auto compatible
+- Automatically updates the screen to follow the active app
 
-Implemented Features
---------------------
+## Recommended compatible apps:
```

Comment:
This heading seems too strong, since it is the same size as the top-level Implemented Features heading.